client/Cargo.toml

```diff
@@ -13,6 +13,7 @@ repository.workspace = true
 [features]
 default = ["strict"]
 strict = ["ldap3_proto/strict"]
+openssl-vendored = ["openssl/vendored"]
```

Comment:
As mentioned you don't need this - you can pass through the openssl vendored flag from the calling Cargo.toml